fighters/demon/src/acmd/tilts.rs

```diff
@@ -78,6 +80,7 @@ unsafe fn game_attackhi3(fighter: &mut L2CAgentBase) {
         WorkModule::on_flag(boma, *FIGHTER_DEMON_STATUS_ATTACK_HI_3_FLAG_CHECK_STEP);
     }
     frame(lua_state, 21.0);
+    FT_MOTION_RATE_RANGE(fighter, 21.0, 30.0, 6.0);
```

Comment:
more frames cut off the end of the move with no reset in motion rate. In your changelog, UTilt1 has no changes listed.